submission-folder/Vina.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * HOMEWORK
+ * rename this file to *your name*.ts
+ * Translate this JS code into valid TypeScript that follows good practices,
+ * following instructions in the comments
+ */
+
+// replace all `var` declarations appropriatly
+
+const NINE_THOUSAND = 9000
+let number = 6
+number = 7
+
+// create a type alias (rmb type alias names use PascalCase)
+type Person = {
+    name: string
+    year: number
+    friends: string[]
+    favouriteletiable: string | number
+}
+const bob : Person = {
+  name: 'Bob',
+  year: 2,
+  friends: ['Steve'],
+  favouriteletiable: 23,
+}
+const steve : Person = {
+  name: 'Steve',
+  year: 1,
+  friends: ["Bob", 'Mary'],
+  favouriteletiable: 'hello', // note that this shall ONLY be string or number
+}
+const mary : Person = {
+  name: 'Mary',
+  year: 1,
+  friends: ["Steve"],
+  favouriteletiable: 100,
+}
+
+
+let students: Person[] = [bob, steve, mary] // assign an appropriate type to `students`
+
+// convert to anonymous function
+// use string literals
+// add appropriate type declarations to function args
+
+//answer
+const printNameAndYear = (student: Person) => {
+    let text = student.name + ' is year ' + student.year 
```

Comment:
you are suppose to change this to [string literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) format